benchmarks/harness/detect.py

```
@@ -0,0 +1,120 @@
+    available: bool
+    reason: str
+
+
+def _pre_commit_available() -> Availability:
+    """Determine whether pre-commit can be invoked.
+
+    Returns:
+        Availability: Status for the pre-commit competitor.
+    """
+    if which("pre-commit") is not None or which("prek") is not None:
+        return Availability(CompetitorTool.PRE_COMMIT, True, "found on PATH")
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)